File: tests/xcalablemp/global-view/gmove/F/chk/chk.f90
subroutine chk_int(ierr)
integer ierr
!$xmp nodes p(*)

!$xmp task on p(1)
  if ( ierr .eq. 0 ) then
     write(*,*) "PASS"
  else
     write(*,*) "ERROR"
     call abort
  endif
!$xmp end task

return
end subroutine

File: tests/xcalablemp/global-view/gmove/C/chk/chk.c
#include <stdio.h>
#include <stdlib.h>

int chk_int(char *name, int ierr){

#pragma xmp nodes p(*)

#pragma xmp task on p(1)
{
  if ( ierr == 0 ) {
     printf("PASS %s\n",name) ;
  }else{
     printf("ERROR\n");
     exit(1);
  }
}
  return 0;

}
